Gemfile

```diff
@@ -1,2 +1,5 @@
 source 'https://rubygems.org'
 gemspec
+
+# needed for now because its not released...
+gem 'motion_print', github: 'OTGApps/motion_print'
```

Comment:
this is in the gemfile because we are using unreleased changes and its the only way I could verify the color forcing, when @markrickert releases the `force_color` stuff we can clean this up.